examples/call_multiple_tools_directly_with_auth.py

```diff
@@ -0,0 +1,175 @@
+"""Example script demonstrating how to call multiple tools directly with authentication.
+
+For this example, we are using the prebuilt Spotify toolkit to start playing similar songs to the currently playing song.
+
+Steps:
+1. Get the currently playing song
+2. Get audio features for the currently playing song
+3. Get song recommendations that are similar to the currently playing song
+4. Start playing the recommended songs
+5. Inform the user which recommended song is now playing
+"""
+
+from typing import Any
+
+from arcadepy import Arcade  # pip install arcade-py
+
+
+# Need to click on a link for every provider
+def get_permissions(client: Arcade, provider_to_scopes: dict, user_id: str) -> None:
+    """Prompt the user to authorize necessary permissions for each provider."""
+    for provider, scopes in provider_to_scopes.items():
+        auth_response = client.auth.start(
+            user_id=user_id,
+            provider=provider,
+            scopes=scopes,
+        )
+
+        if auth_response.status != "completed":
+            print(f"Click this link to authorize: {auth_response.authorization_url}")
+            input("After you have authorized, press Enter to continue...")
+
+
+def call_tool(client: Arcade, tool_name: str, user_id: str, inputs: str | dict = "") -> Any:
```

Comment:
Do you ever need to pass a `str` to `inputs`? (looks like it is only `dict` in this file)